enginetest/testdata.go

```diff
@@ -104,6 +104,28 @@ func createSubsetTestData(t *testing.T, harness Harness, includedTables []string
 		})
 	}
 
+	if includeTable(includedTables, "geometry_table") {
```

Comment:
I should have brought this up in an earlier PR, but:

The geometry tables should not be a part of the standard test setup for every engine test. The engine tests that require geometry types should install these tables separately.